tools/block-generator/run_runner.sh

```diff
@@ -54,4 +54,6 @@ $(dirname "$0")/block-generator runner \
 	--test-duration 30s \
 	--log-level trace \
 	--postgres-connection-string "host=localhost user=algorand password=algorand dbname=generator_db port=15432 sslmode=disable" \
-	--scenario ${CONFIG}
+	--scenario ${CONFIG} \
+	--db-round ${2:-0} \
```

Comment:
Could this be read from the db at startup time to avoid needing to provide?

Reply:
yes, but it seems expensive to create a connection just to read 1 number.

Reply:
Ok. What's the exact round number I should supply? I can add that to the `Makefile` I'm introducing in my followup #5394

Reply:
`RunnerCmd.Flags().Uint64VarP(&runnerArgs.NextDBRound, "next-db-round", "n", 0, "next round on db. It is the next_account_round from Indexer's metastate table.")`

Reply:
yes, it's not working with I provided a scenario directory instead of a scenario file path. I'll fix this in a follow up PR.